dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -65,15 +65,15 @@ sources:
                       AND deactivat IS NULL
                       AND class NOT IN ('201', '213', '218', '219', '220', '221', '224', '225', '236', '240', '241', '290', '294', '297')
                   meta:
-                    description: attic (attic type) should not be null
+                    description: attic (Attic Type) should not be null
```

Comment:
Where a field is visible in iasWorld, we now match the iasWorld capitalization scheme in its description.